### ISSUE TITLE
Donation / HONEY button

### ISSUE THREAD

**Issue by yeqbfgxjiq** [1Hive/1Hive#17]
As soon as the [Token Request App](https://github.com/1Hive/ideas/issues/3) is ready we should feature it on the website with an explanation and link so that people can donate/contribute by buying HONEY.

**Comment by lkngtn** [comment]
I think this may be related to https://github.com/1Hive/ideas/issues/3

**Comment by yeqbfgxjiq** [comment]
Good point. Just updated the initial comment to reflect that

**Comment by yeqbfgxjiq** [comment]
It sounds like we're not keen on building the Token Request App right away. In that case, does it make sense to just create a blurb in one of the sections on the homepage telling people what HONEY is and that if they want to engage with BEEs and support project they can send ANT/DAI to our vault and we'll send them back some HONEY?
- if so, BEEs would need to agree on an exchange rate until we have a bonding curve
- then we would just need to agree to vote to mint and send HONEY to people when they throw stuff into our vault 

EDIT: 
Starting coding a draft of what this might look like on the website and realized that saying: "Send us ETH. Don't worry... we'll send you some tokens back" is the most sketch scammy way to exchange tokens.  Definitely NOT a good idea! lol
Maybe instead we could just open up a pool on UniSwap? It would require HONEY and ETH setup (it's a bring your own liquidity model), but at least that would get us started and make the token exchange verifiable and automatic.

**Comment by lkngtn** [comment]
I think best solution to accept donations is to just use and link to gitcoin.co/grants for now. (there is no expectation of tokens in return so there is nothing extra we need to do). And to solicit donations/submit grant applications directly where appropriate. 

For exchanging Honey for DAI, we could either decide to move forward with the token request app (I'm not opposed to this btw, just seemed like it warranted some more thought & discussion as its not immediately obvious that it is something we want to pursue). 

Or start looking at the Apiary contracts and start contributing to documentation there--I'm meeting with olivier from Aragon black tomorrow morning to sync up on Apiary so may have additional information on their progress and how we might be able to contribute / accelerate things.

**Comment by yeqbfgxjiq** [comment]
Ok cool. I think that makes sense. 
- grants => vault now
- token request app in the near future if everyone wants to work on it
- apiary as the goal because it solves all our problems and more, but will take a min to develop

If we got the token request app built then the Aragon Black team could potentially throw us a few coins for helping out on the Apiary/Fundraising app... just sayin...

**Comment by lkngtn** [comment]
> If we got the token request app built then the Aragon Black team could potentially throw us a few coins for helping out on the Apiary/Fundraising app... just sayin... 

I don't think that is in their budget, and if we plan on using it ourselves I feel like this is one of the things where we should just take the initiative and help out as it will help us learn how we want to implement it for 1hive.

**Comment by yeqbfgxjiq** [comment]
I didn't say it had to be a huge kickback. Just a little something to share the love and put some fuel into the hive. Even just getting tokens of appreciation and testing out the model is exciting.

Also, I agree that it makes sense to work on Apiary regardless because it's awesome and we're going to use it. I'm just saying that it would be cool if we started pushing that whenever we create value a little value/recognition gets returned to the hive to be shared with the BEEs as well 🍯

**Comment by yeqbfgxjiq** [comment]
So until the Token Request App or Apiary is built, I'd say we just go ahead and put the main DAO vault address on a "donations" page. This could be displayed in the More section of the footer with the title "donate". Nothing fancy, just an address people can donate to if they want to support the project. Then we can upgrade this as we design better and better apps. 
ATM this would have a **HUGE WARNING SIGN** that this is Rinkeby so we can't accept mainnet ETH, but in the future then we could change that on the website easily.
@lkngtn @0xGabi Does that sound good?

**Comment by lkngtn** [comment]
We can create a mainnet dao with just bees and voting and a vault for now.

**Comment by yeqbfgxjiq** [comment]
> We can create a mainnet dao with just bees and voting and a vault for now.

@lkngtn are you saying that TPS is ready for mainnet?

**Comment by 0xGabi** [comment]
I think what Luke was referring is create a really simple DAO on mainnet where we can store the founding in the meantime.

**Comment by burrrata** [comment]
Oh! Yeah I'm down with that too. Just a good old multi-sig?

**Comment by yeqbfgxjiq** [comment]
<s>Started creating a multi-sig DAO to accept donations, but [mainnet.aragon.org/#/](https://mainnet.aragon.org/#/) is telling me that I need "at least 0.1 ETH." How much does it cost to setup a mainnet Aragon DAO?</s>

NVM, looks like we already have https://mainnet.aragon.org/#/1hive.aragonid.eth/
Thanks @lkngtn :)

**Comment by yeqbfgxjiq** [comment]
@lkngtn to confirm, if someone sends tokens to the finance app (0x5ad3c83c49d5f0e0c2a3b3520bf2170c3272ffa8) that's a direct deposit to the DAO vault right?
- and yes I just tried it and it worked, but I want to be extra sure before putting it on the website lol
